popSimManu.tex

```diff
@@ -592,7 +598,7 @@ \subsection*{Use case: comparing methods of demographic inference}
 Africa from which a European population splits off following a bottleneck, with no 
 post-divergence gene flow between the African and European population (Figure \ref{fig:two_popn_fly}A).
 Here again, we find that \dadi and \fastsimcoal infer more consistent histories,
-but ignore the brief population bottleneck in Europe. In addition, \dadi
+but but do not detect the brief bottleneck in Europe, likely due to not including that epoch in the inference. In addition, \dadi
```

Comment:
"not including in the inference" is a big confusing - what do we mean here?

Reply:
i think what is meant is that there isnt a separate epoch in the dadi or fsc model to account for a bottleneck? how would you suggest better stating that?

Reply:
how about
> infer more consistent histories but do not include Europe's brief bottleneck in their model.
... ?

Reply:
hmm this doesnt seem to quite convey the point that the inference ignores it because the model assumed a priori doesnt really allow for the model to separately infer a bottleneck pop size parameter. how about: 
"but do not detect the brief bottleneck in Europe, likely due to the constraints of the inference model."

Reply:
Hm, ok. Can this be more definitive? If the models just plain don't allow a bottleneck, then it's not "likely" due to the constraints, it's definately because of the constraints; and we should say *what* constraints (ie, the models just don't have a bottleneck)?